dimod/discrete/discrete_quadratic_model.py

```diff
@@ -867,8 +872,9 @@ def to_file(self, compress=False, compressed=None, ignore_labels=False,
         # the section containing most of the data, encoded with numpy
         if compressed is not None:
             warnings.warn(
-                "Argument 'compressed' is deprecated and in future will raise "
-                "an exception; please use 'compress' instead.",
+                "Argument 'compressed' is deprecated since dimod 0.9.9 "
+                "and will be removed in 0.12.0. "
+                "Use 'compress' instead.",
```

Comment:
I think we have this in dwave-system too, for ``sample_dqm``. Will that be updated too?

Reply:
Yes, though I don't think it's super urgent since dwave-system [handles it separately](https://github.com/dwavesystems/dwave-system/blob/4fa4b27a422a63afa65e90773ed4cf4d918c2f9f/dwave/system/samplers/leap_hybrid_sampler.py#L488).